integrationTests/vm/txsFee/guardAccount_test.go

```diff
@@ -828,6 +833,50 @@ func TestGuardAccount_Scenario1(t *testing.T) {
 	}
 	testGuardianStatus(t, testContext, alice, expectedStatus)
 
+	// 12. alice sends a malformed set guardian transaction, should not consume gas
+	// 12.1. transaction with value
+	nonceAlice := getNonce(testContext, alice)
+	setGuardianData := []byte("SetGuardian@" + hex.EncodeToString(bob) + "@" + hex.EncodeToString(uuid))
+	tx := &transaction.Transaction{
+		Nonce:    nonceAlice,
+		Value:    big.NewInt(10),
+		RcvAddr:  alice,
+		SndAddr:  alice,
+		GasPrice: gasPrice,
+		GasLimit: setGuardianGas + transferGas,
+		Data:     setGuardianData,
+	}
+	returnCode, err = processTransaction(testContext, tx)
+	require.Equal(t, vmcommon.UserError, returnCode)
+	require.NotNil(t, err)
+	require.ErrorIs(t, err, process.ErrTransactionNotExecutable)
+	newNonceAlice := getNonce(testContext, alice)
+	// tx not executed
+	require.Equal(t, nonceAlice, newNonceAlice)
+
+	// 12.2. too many parameters in data, last one not hex (tx with notarization not builtin func call)
+	nonceAlice = getNonce(testContext, alice)
```

Comment:
this is not required, we can compare with the original value, there is no way it can pass the asserts and change its nonce

Reply:
removed, it was just made as pair with what was checked at the end, for clarity.